srcs/runners_manager/vm_creation/CloudManager.py

```diff
@@ -70,6 +70,10 @@ def delete_vm(self, runner: Runner):
     def delete_images_from_shelved(self, name):
         raise NotImplementedError
 
+    @abc.abstractmethod
+    def add_labels_to_instance(self, instance_name: str, labels_webhook: dict):
```

Comment:
What happens to the other class that don't have this method implemented? might need to add a dummy function

Reply:
I tried something! Tell me if it's better

Reply:
I would go for either `update_vm` or `update_vm_metadata`, here's why:
* I like the word instance better, but the rest of the code use `vm` (on the function names, the variables use instance, it's bit of a mix and match I see 😅)
* Identifiers is accurate but I don't see it used very often, metadata is more common at least for me.
* Technically we're adding some labels but we're updating others